app/javascript/designSystem/__stories__/avatarsAndLogos.stories.jsx

```diff
@@ -0,0 +1,131 @@
+import { h } from 'preact';
+import { storiesOf } from '@storybook/react';
+
+import './typography.scss';
+
+storiesOf('Base/Components', module).add('Avatars & Logos', () => (
```

Comment:
I was thinking that "Components" will not belong to "Base", instead they would be its own first-level category.
```
Base
- Typography
- Colors
- Spacing
Utility Classes
- Margins & Padding
- Flexbox
- Grid
- Positioning
- Responsiveness
- ...
Components
- Avatars and Logos
- Buttons
- Dropdowns
- Forms
-...
```

Reply:
That makes sense. I put Components under base because we currently have a top-level Components already which is application components, e.g. `<Article />`. It's definitely easy to move this stuff around. Maybe stuff like `<Article />` should be classified as Application Components.